config/katello-answers.yaml

```
@@ -91,7 +91,8 @@ foreman_proxy::plugin::pulp:
 foreman_proxy::plugin::remote_execution::ssh: false
 foreman_proxy::plugin::salt: false
 foreman_proxy_content: true
-katello: true
+katello:
+  user_pulp_2_for_yum: false
```

I'm a bit confused, this is already the default value. The goal is new installs should use Pulp 3 for yum content and the module defaults to this (https://github.com/theforeman/puppet-katello/pull/347/files#diff-481a583a7371c0062cb0fcb5c039626cR22). The migration is needed to ensure upgrades continue to use Pulp 2 for yum content.

---

Note the typo as well (which will be silently ignored):
```suggestion
  use_pulp_2_for_yum: false
```

---

Ah, for some reason I convinced myself that the module would default to true and didn't double check.

OK, I'll re-write it as a separate migration this time.